allure-junit5/src/test/java/io/qameta/allure/examples/junit5/AllureParameterizedTest.java

```diff
@@ -12,13 +12,11 @@
 public class AllureParameterizedTest {
 
     @ParameterizedTest(name = "{displayName} [{argumentsWithNames}]")
-    @ValueSource(strings = {"First Name", "Second Name"})
+    @ValueSource(strings = {"John", "Mike"})
     @DisplayName("allureParameterizedTest displayName")
     @Description("allureParameterizedTest description")
```

Comment:
We have a separate `AllureDescriptionTest`. Why to overcomplicate this already complex example with one more annotation that do not reflect the "parametrization" context in any way?

Though, we could change it to something like:

```java
    @ParameterizedTest(name = "{displayName} [{argumentsWithNames}]")
    @ValueSource(strings = {"John", "Mike"})
    @DisplayName("Custom displayName for allureParameterizedTest")
    @Description("The display name of this test will be built in two steps:\n"
            + "1. We tune the default 'allureParameterizedTest' display name \n"
            + "   to 'Custom displayName for allureParameterizedTest'\n"
            + "2. Then we add ' [{argumentsWithNames}]' to it")
    public void allureParameterizedTest(String name) {
        parameter("Name", name);
    }

```

Yet it seems pretty complicated to me...